<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>utconnectorClass</class>
 <widget class="QWidget" name="utconnectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>243</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>utconnector</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="welcomeLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Welcome to &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;UrTConnector&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;UrTConnector simply allows you to connect to UrbanTerror game server. &lt;span style=&quot; color:#aa0000;&quot;&gt;Have fun!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="welcomeLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ioexeLabel">
       <property name="text">
        <string>Urban Terror executable:</string>
       </property>
       <property name="buddy">
        <cstring>ioexeEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ioexeEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ioexeButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="serverLabel">
       <property name="text">
        <string>Server:</string>
       </property>
       <property name="buddy">
        <cstring>serverEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="serverLabel_2">
       <property name="text">
        <string>Example: 195.151.227.134:27960</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
       <property name="buddy">
        <cstring>serverEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="playernameLabel">
       <property name="text">
        <string>Playername:</string>
       </property>
       <property name="buddy">
        <cstring>playernameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="playernameEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="passwordCheckBox">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ioexeButton</tabstop>
  <tabstop>ioexeEdit</tabstop>
  <tabstop>serverEdit</tabstop>
  <tabstop>playernameEdit</tabstop>
  <tabstop>passwordCheckBox</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>utconnectorClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>passwordEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
